app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,10 @@
+<section class = "login">
```

Comment:
This is small nitpicking, but HTML/CSS convention is to not have spaces between, so: `class="login"`